db-init/seed.sql

```diff
@@ -0,0 +1,24 @@
+-- Seed log records table
+INSERT INTO log_records (
+  employee_id, 
+  resident_first_name, 
+  resident_last_name, 
```

Comment:
this needs to be updated to resident_id but we need to wait for #50 to be merged first

Reply:
it can never be escaped

Reply:
we can ignore this and merge what you have for now